shared/types/proto/tx.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package shared;
+
+option go_package = "github.com/pokt-network/pocket/shared/types";
+
+message DefaultTxResult {
+  bytes tx = 1;
+  int64 Height = 2;
+  int32 index = 3;
+  int32 result_code = 4;
+  string error = 5;
```

Comment:
Should we have a TODO to have a type for errors or result_code?

Reply:
No, actually considering our block correctness design, both of these should be deprecated shortly. Will add TODO for that

Reply:
We have type for errors and code in the utility package. Will use those when moved